packages/core/src/html/NodeProcessor.js

```diff
@@ -1,6 +1,7 @@
 const path = require('path');
 
 const cheerio = require('cheerio');
+const lodashHas = require('lodash/has');
```

Comment:
We already have this require several lines below as `_.has`. Let's use that to minimise duplication

Reply:
I see, I didn't catch that earlier, thank you!